File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        //int answer = 0;
        int a = 1;
        int b = 0;

        for(int num : num_list){
            a *= num;
            b += num;
        }
        if(a < b*b) {
            return 1;
        }else{
            return 0;
        }
        //return answer;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    answer = 0
    multiplication = 1
    squared = 0

    for i in range(len(num_list)):
        multiplication *= num_list[i]
        squared += num_list[i]
        
    answer = 1 if multiplication < squared ** 2 else 0
    
    return answer